src/main/kotlin/io/github/orangain/prettyjsonlog/Extract.kt

```diff
@@ -108,11 +108,20 @@ fun extractLevel(node: JsonNode): Level? {
     }
 }
 
-private val messageKeys = listOf("message", "msg")
+private val messageKeys = listOf("message", "msg", "error.message")
 
 fun extractMessage(node: JsonNode): String? {
     return messageKeys.firstOrNull { node.has(it) }?.let { messageKey ->
         node.get(messageKey)
             ?.asText()
     }
 }
+
+private val stackTraceKeys = listOf("stacktrace", "error.stacktrace", "error.stack_trace")
```

Comment:
I understand that EcsLayout.json uses the `error.stack_trace` key, but could you tell me where the other keys `stacktrace` and `error.stacktrace` come from?

If there is a JSON format that uses such keys, I would like to add test cases for them. Otherwise, I would like to remove them from the `stackTraceKeys`.

Reply:
ok lets remove them